src/sections/FAQSection/FAQSection.tsx

```diff
@@ -11,11 +10,27 @@ import { List, ListItem } from '../../components/List';
 import { SectionLayout } from '../../components/SectionLayout';
 import PlusIcon from '../../images/svg/add-black.svg';
 
+const FAQContainer = styled(Container)`
+  && {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+`;
+
 const MaxWidth = styled.div`
-  max-width: 500px;
+  max-width: 700px;
+`;
+
+const FAQTitleContainer = styled.div`
+  && {
+    vertical-align: middle;
+  }
 `;
 
 const Topic = styled.p`
+  color: ${({ theme }) => theme.color.black};
```

Comment:
Okey... now I see the `black` is defined in the theme as `'#00141E'`. I quite don't like it. Very confusing.

Reply:
For me, this is the only blocking comment.

Reply:
I changed it to `stormyBlue` 👌🏻